teleport/tests/test_common.py

```diff
@@ -11,11 +11,13 @@
 pytestmark = [pytest.mark.unit]
 
 
-def test_connect_exception(dd_run_check):
+def test_connect_exception(dd_run_check, caplog):
```

Comment:
You are not asserting `health.up` metric value is 0 and it contains `teleport_status:unreachable` tag

Reply:
Done!